<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1601689408148' stop='1601689456522'>
    <name>Connection</name>
    <title>Connection</title>
    <test-cases>
        <test-case start='1601689414917' status='failed' stop='1601689450790'>
            <name>In-place</name>
            <title>In-place</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1601689414919' status='undefined'>
                    <name>Can start in-place connection</name>
                    <title>Can start in-place connection</title>
                    <attachments>
                        <attachment title='Test failed: Connection :: In-place' source='7e22ff90-9b5b-490a-ae86-cc66f97f342f-attachment.png' type='image/png' size='72095'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TimeoutError: waiting for selector `button#approve` failed: timeout 30000ms exceeded</message>
                <stack-trace>TimeoutError: waiting for selector `button#approve` failed: timeout 30000ms exceeded
                    at new WaitTask (/home/travis/build/Automattic/jetpack/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:411:34)
                    at DOMWorld.waitForSelector (/home/travis/build/Automattic/jetpack/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:319:26)
                    at Frame.waitForSelector (/home/travis/build/Automattic/jetpack/node_modules/puppeteer/lib/cjs/puppeteer/common/FrameManager.js:842:51)
                    at waitForSelector (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:33:30)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>